Exoft.Gamification.Api.Common/Models/AchievementModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Exoft.Gamification.Api.Common.Models
+{
+    public class AchievementModel : BaseModel
```

Comment:
Implementation of model is wrong. Please remove it for now.... There will be task that describes everything regarding CRUD of achievements